torchvision/prototype/datasets/decoder.py

```diff
@@ -13,3 +16,17 @@ def raw(buffer: io.IOBase) -> torch.Tensor:
 
 def pil(buffer: io.IOBase, mode: str = "RGB") -> torch.Tensor:
     return pil_to_tensor(PIL.Image.open(buffer).convert(mode.upper()))
+
+
+def av_kf(buffer: io.IOBase, **read_video_kwargs: Any) -> Dict[str, Any]:
+    with unittest.mock.patch("torchvision.io.video.os.path.exists", return_value=True):
```

Comment:
I don't think we need that anymore. I only added it in #4541 because I used `torchvision.io.video.read_video` and it enforces that the input is a file on disk although it can very well work with file handles.